feat(content): Adelita Journalism missions 1 through 5 (adapted from #5112) (#9530)
endless-sky/endless-sky: data/human/human missions.txt

	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
	minor
	source
		attributes "paradise"
		not attributes "mining"
		not attributes "textiles"
		not attributes "research"
		not attributes "factory"
	destination "Moonshake"
	cargo "exhibition installations" 5
	to offer
		random < 80
		has "Expedition to Hope 2: done"

	on offer
		conversation
			`Walking around <origin>'s luxurious spaceport, you notice a large group of people gathered around one of the larger hangars; there are large signs that say "The Evacuation of Hope" in bold letters by it. It looks like some sort of exhibition.`
			choice
				`	(Check it out.)`
				`	(Not right now.)`
					defer

			`	You approach some of the people at the end of the line and ask what's going on. "It's the last day of Adelita's exhibition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their exhibitions are always free so this is a great chance to see it if you haven't already."`
			choice
				`	"Sure, why not."`
				`	"Maybe some other time."`

> 					defer

Author: I also had problems with this, but I didn't want to just stick a decline there. I'll think of some way to change it.

> 					defer

Reviewer: I think it's good as is with last week here in the intro. If we do come back, it just ends up that we could have seen it not on the very last day if we had gone the first time.

> 			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`

Author: Oxford Dictionary (via Apple) says both media and mediums are fine.

> 			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`

Reviewer 2: https://www.merriam-webster.com/dictionary/medium says it's usually media, though that does mean mediums is ok.

			`	"It's more of a collage than an exact rendering," says the young woman you met in line. You were so distracted by the experience you didn't notice her approach. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. So what do you think?"`
			choice
				`	"Impressive."`
				`	"It's OK."`

			`	"Well, it was just as I imagined it," the woman replies. "Oh! I have an exam to cram for. Enjoy the expo, I think it closes soon," she says as she heads for the exit.`
			choice
				`	(Continue wandering around the exhibit.)`
				`	(Go to the exit.)`
					goto exit

			branch katya
				has "FW Katya 4: done"

> 			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. In the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`

Author: ```suggestion
			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. From the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`
```

				goto "move on"

			label katya

> 			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. In the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`

Author: ```suggestion
			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships; it strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. In the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`
```
To keep it consistent with the other suggestion?

			choice
				`	"I would love to."`
				`	"Well, why not?"`
				`	"Sorry, I have more pressing matters at hand."`
					goto decline

			`	"Great!" Mariel says. "Bring your ship to this hangar, and I will have our crew get everything ready to load."`
				accept

			label decline
			`	"Oh, I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call," Mariel says, but then she glances at her watch. "Well anyway, the exhibit is closed. I'm going to have to ask you to leave."`
				decline

			label exit
			`	Following where the woman went, you find the true exit of the hangar and go back out onto the familiar spaceport tarmac.`
				decline

	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`

	on complete
		payment 80000

> 		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`

Author: I think other missions have their logs in `on complete`, but I will check later.

> 		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`

Author: You're correct.

			choice
				`	"In."`
				`	"Out."`
					goto out

			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories, you will be going with our associate Bunker to spy on the Syndicate's rendezvous with the pirates on <planet>."`

> 			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro and we have an array of long range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`

Author: I don't remember what our stance is on dashes in American English, I thought it was more of a British English thing? I'll add the dash, but I want to make sure for the style guide.

> 			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro and we have an array of long range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`

Author: ```suggestion
			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`
```

			`	You go back to the <ship>'s landing pad and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
			choice
				`	"I am Captain <first> <last>, well met."`
				`	"Hi, I'm <first>."`
					goto loadup
			`	They all frown at your formality.`

			label loadup
			`	"Good. Now that that's done with, let's load up," says Bunker. "Taking down the Man. Whoo, I love it!"`
				accept

			label out
			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`
				decline

	on visit
		dialog `The filming equipment is aboard an escort. Better take off and wait for it to arrive.`

mission "Adelita 3"
	landing
	name "Film slave transaction"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
	source "Maker"
	destination "Hippocrates"
	"apparent payment" 200000
	infiltrating
	to offer
		has "Adelita 2: done"
	passengers 1
	cargo "stealth drones" 1
	on offer
		conversation

> 			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck, and the two walk off into the spaceport. You and Bunker go back to the <ship> and prepare to take off for <planet>.`

Reviewer 3: ```suggestion
			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take off for <planet>.`
```

				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

> mission "Adelita 4"

Author: Is that possible? It would make sense, but you could also just imagine the rest of your ships are hanging around in orbit or something. But I don't know if you can alter dialogue to say that either.

> mission "Adelita 4"

Reviewer 2: > Is that possible?

It definitely is, Hurleveur does that a lot in the unfettered campaign PR.

> imagine the rest of your ships are hanging around in orbit or something. But I don't know if you can alter dialogue to say that either.

Yeah, maybe a branch mentioning the rest of your ships would work. `You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space, away from prying eyes.` or something.

	landing
	name "Slave trade"
	description "Pick up the team on <destination>. (You will be paid <payment> after you complete the entire job.)"
	source "Hippocrates"
	destination "Maker"
	"apparent payment" 200000
	to offer
		has "Adelita 3: done"
	passengers 1
	cargo "stealth drones" 1

	on offer
		conversation

> 			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but as you are well within the atmosphere Bunker releases the drones from the open cargo bay without needing a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`

Reviewer 3: ```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on auto-pilot to the rendezvous site."`
```

Author: ```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

> 			`	Once the drones are on site you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`

Reviewer 3: ```suggestion
			`	Once the drones are on site, you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
```

> 			`	He hums to himself for awhile before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them. But for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`

Reviewer 3: A while versus awhile in ES files: 206 results versus 0 results.
```suggestion
			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`
```

			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

> 			`	"And you know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`

Reviewer 3: ```suggestion
			`	"And you know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`
```

				goto showtime

			label plan

> 			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`

Reviewer 3: ```suggestion
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`
```

			label showtime

> 			`	"Show time!" he says, rubbing his hands together. Both of you watch as a capital ship in pirate colors lands near the security detail, probably deafening them with the sound of their repulsors. It's accompanied by a small fleet of Quicksilver escorts. As soon as the loading ramp is down the security detail gets to work herding the slaves from one ship to another; meanwhile, the detail hit and shove anyone that moves too slowly for their liking. It's hard to watch.`

Reviewer 3: ```suggestion
			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship in pirate colors lands near the security detail, probably deafening them with the sound of their repulsors. Accompanying it is a small fleet of Quicksilver escorts. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; the detail hit, shove, and shout at anyone that moves too slowly for their liking.`
```

> 			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. "What do we have here?" he says. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky, as if he sees something. He motions and the Quicksilver escorts fire up their engines and begin taking off.`

Reviewer 3: ```suggestion
			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky, as if he sees something. He motions, and the Quicksilver escorts fire up their engines.`
```

			`	"Oh, I don't like the look of this," says Bunker.`
			choice

> 				`	"I'm sure it has nothing to do with us. Keep filming."`

Reviewer 3: ```suggestion
				`	"Keep filming."`
```

				`	"Call back the drones immediately."`
					goto "call drones"

> 				`	"I'm sorry, but we'll have to leave the drones behind."`

Reviewer 3: ```suggestion
				`	"I think we'll have to leave the drones behind."`
```

					goto "no can do"
				`	"Did he see us?"`
					goto "see us"

			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
			choice
				`	"That's an order!"`

> 				`	"OK. Call the drones back."`

Reviewer 3: ```suggestion
				`	"Okay. Call the drones back."`
```

					goto "call drones"

			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`

> 			`	You notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers, who have found your location and are now speeding towards you. You try to pull out to space but another flotilla of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`

Reviewer 3: ```suggestion
			`	You notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers. You try to pull out to space but another flotilla of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`
```

				die

			label "see us"
			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`
			choice
				`	"Call back the drones immediately."`
					goto "call drones"

> 				`	"I'm sorry but we have to leave the drones behind."`

Reviewer 3: ```suggestion
				`	"I think we have to leave the drones behind."`
```

			`	The moment you pull out of the canyon, pirate Quicksilvers appear in the distance from all directions. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
				launch

			label blaster
			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says. You turn around to look for the handcuffs when you notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers, who have found your location and are now speeding towards you. You try to pull out to space but another flotilla of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`
				die

	npc
		personality heroic nemesis vindictive launching
		government "Pirate"
		ship "Quicksilver" "Don't Run"
		ship "Quicksilver" "Obey"
		ship "Quicksilver" "Work Is Freedom"
		ship "Quicksilver" "Do As You're Told"
		ship "Quicksilver" "Got You"

> 		ship "Quicksilver" "Futile"

Author: Yeah I left this from the original PR.

> 

Reviewer 3: ```suggestion
		on disable
			set "adelita: killed pirates"

```

	on visit
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

mission "Adelita 5"
	name "Slave trade"

> 	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"

Reviewer 2: Honestly, I'm not entirely sure if this will display the payment in the `on complete` branch or the apparent payment.

Author: I thought the apparent payment would come first, but I'll test it later.

Reviewer 2: It depends on what part of the mission you are in... it's a bit messy when you have both. The wiki says

> `<payment>` = "1 credit" or "N credits," where N is the payment amount from the on complete block (or, beginning in v. 0.10.0, the "apparent payment" of the mission, if one is given), unless the replacement is in an on * block or below a conversation apply node that has its own payment, in which case that on * block's or the most recent apply's payment is used

If I read that correctly, that would indeed show the apparent payment in the description. But yeah, this is complex enough to need testing.

~~I added the damn thing, and I'm still not sure how the fallback thing works exactly.~~

Author: <img width="1440" alt="Screenshot 2023-11-09 at 11 32 01 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/acee2b8f-4cfc-4a47-b764-a1c07618c4cb">
It works.

	minor
	source "Maker"
	destination "Moonshake"
	"apparent payment" 200000
	landing
	to offer
		has "Adelita 4: done"
	passengers 3
	cargo "stealth drones" 1

	on offer
		conversation
			branch experienced

> 				"combat rating" > 100

Reviewer 3: ```suggestion
				has "adelita: killed pirates"
```

> 			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed- both emotionally and literally.`

Reviewer 3: ```suggestion
			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
```

> 			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed- both emotionally and literally.`

Reviewer 2: ```suggestion
			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
```
I think this is the style we use in the game.

			`	"That was horrible, but we got it done. Can we leave now?" asks Ale.`
				launch
			label experienced

> 			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed- both emotionally and literally.`

Reviewer 3: ```suggestion
			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
```

> 			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed- both emotionally and literally.`

Reviewer 2: ```suggestion
			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
```

			`	"That was awful. Can we leave now?" asks Ale.`
				launch

	on visit

> 		dialog `You left the <cargo> behind. Better take off and wait for them to enter the system.`

Reviewer 3: ```suggestion
		dialog phrase "generic cargo and passenger on visit"
```

	on complete
		payment 220000

> 		log `Helped the underground journalistic collective Adelita Zero gather video evidence of Syndicate participation in the slave trade. Had to escape pirates while doing so.`

Reviewer 3: ```suggestion
		log `Helped the underground journalistic collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade. Had to escape pirates while doing so.`
```

		conversation

> 			`Ale, River and Bunker look glad to be back on <planet>. "Ugh. We saw such horrible things," says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you. We'll let you know when it's done. Also, we might need you for some future projects, so expect us to be in touch." The group leaves your ship.`

Reviewer 3: ```suggestion
			`Ale, River, and Bunker look glad to be back on <planet>. "Ugh. Being there in person was bad enough," says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you. We'll let you know when it's done. Also, we might need you for some future projects, so expect us to be in touch." The group leaves your ship.`
```

			label "move on"
			`	As you move on, you're startled to find yourself face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`
			`	"Oh, how rude of me," she says. "My name is Mariel, I organize these exhibitions and act as a spokesperson for Adelita. And you are?"`
			choice
				`	"Captain <last>."`

> 			`	"Captain," she says, her eyes lighting up. "Well as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`

Reviewer 4: I would keep this comma, and using ‘also’ changes the meaning slightly. If she says ‘now late’, you get the impression that she had been waiting for him and so is frustrated with him, using ‘also’ makes it seem more like a list of general failings which may or may not have any bearing at that moment. I think I’d stick with the original.

> 			`	"Captain," she says, her eyes lighting up. "Well as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`

Author: ```suggestion
			`	"Captain," she says, her eyes lighting up. "Well, as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`
```

			`	After a couple hours, the whole exhibition is packed up and ready for transport to <planet>.`
				accept

			label decline
			`	"Oh. I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call." Mariel glances at her watch. "Well anyway, the exhibit is closed. I'm going to have to ask you to leave."`
				decline

			label exit
			`	Following where the woman went, you find the true exit of the hangar and go back out onto the familiar spaceport tarmac.`
				decline

	on accept
		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`

	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`

	on complete
		payment 80000
		event "adelita 2 offer expired" 1
		dialog
			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of crew walks up to you and hands you a note. It reads:`

> 			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. - Adelita Zero"`

Reviewer 4: I agree that ‘doing’ doesn’t sound quite right, but ‘partaking in’ sounds too formal and slightly archaic. Other options could be ‘carrying out’, ‘engaging in’, ‘performing’, or ‘actioning’ but I don’t like any of them. I would stick with ‘doing’ unless somebody has a better suggestion.

> 			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. - Adelita Zero"`

Author: ```suggestion
			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed. - Adelita Zero"`
```

event "adelita 2 offer expired"

mission "Adelita 2"
	name "Film Syndicate sweatshop on <planet>"
	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"
	cargo "stealth drones" 1
	passengers 3
	"apparent payment" 200000
	source "Moonshake"
	destination "Maker"
	to offer
		has "Adelita 1: done"
		not "adelita 2 offer expired"

	on offer
		conversation

> 			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat. Before your meal arrives, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`

Author: ```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`
```

> 			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`

Author: ```suggestion
			`	"Sorry for the cloak and dagger bit," the woman says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`
```

			choice
				`	"I found it quite moving."`
				`	"It was OK."`
					goto wasok
				`	"It was too political for me."`
					goto political
			`	"I'm glad you liked it," she says, smiling.`
				goto choice

			label wasok
			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says, smiling.`
				goto choice

			label political
			`	"Oh. Unfortunately for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installations," she says with a smile. She gets her smoothie to-go and walks out.`
				decline

			label choice

> 			`	"Mariel said you saw me take the photos of the woman and her baby," she says. "I really like that moment, but not many notice it." She leans in closer. "We need a captain to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows. Oh, and we have to leave today. It will pay, of course. <payment>. So what do you say? Are you in or out?" As she finishes, your order is served while she drinks her smoothie.`

Author: ```suggestion
			`	"I head from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She gives a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, then pick up the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`
			`	"Oh, and we have to leave today. It will pay, of course. <payment>. So what do you say? Are you in or out?" As she finishes, your order is served while she drinks her smoothie.`
```

			choice
				`	"In."`
				`	"Out."`
					goto out

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop. Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories, you will be going with an associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`

Author: ```suggestion
			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop.
			`	"Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction.
			`	"So while we are getting the footage we need in the factories, you will be going with an associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
```

> 			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`

Author: ```suggestion
			choice
				`	"Hold on, pirates?"`
				`	"Won't this get me in trouble with the Syndicate?"`
					goto syndicate
			`	"Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But you might encounter some lone pirates on the way, so you'll need to be quick."`
				goto leave

			label syndicate
			`	"Well, I did say we had enemies in high places, didn't I? But don't worry, Bunker's a pro. We have an array of long-range stealth drones that do the spying, you won't be anywhere near them. Though pirates do frequent the system, so you'll need to be quick."`

			label leave
```

			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker.`
			choice
				`	"I am Captain <first> <last>. Well met."`
				`	"Hi, I'm <first>."`
					goto loadup

> 			`	They all frown at your formality.`

Author: What about this? It's a little simple, maybe.
```suggestion
			`	They all frown at your formality. "Well, let's load up," says Bunker.`
				accept
```

			label loadup
			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
				accept

			label out

> 			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`

Author: ```suggestion
			`	"I'm sorry to hear that... I suppose it just wasn't meant to be." She gulps down her smoothie at lightning speed before leaving just as quickly.
```

				decline

	on visit
		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`

mission "Adelita 3"
	name "Film slave transaction"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
	landing
	cargo "stealth drones" 1
	passengers 1
	infiltrating
	"apparent payment" 200000
	source "Maker"
	destination "Hippocrates"
	to offer
		has "Adelita 2: done"

	on offer
		conversation

> 			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to depart for <planet>.`

Author: ```suggestion
			`You drop off Ale and River at the spaceport, watching as they try not to seem either visibly embarrassed to be in their security uniforms, or uncomfortable with their disguises, even though they're now virtually indistinguishable as Syndicate personnel; Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
```

				goto "on site"

			label "player has a fleet"
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

			label "on site"
			`	Once the drones are on site, you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`

> 			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`

Author: I don't really see what's wrong with it? Or do I just say he turns to you and starts talking?

> 			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`

Reviewer 5: Turn and start talking would be better, I think.

> 			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`

Reviewer 6: I do think he should just turn to the player and start talking.

			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

> 			`	"And you know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`

Author: ```suggestion
			`	"And you know they'll use those poor people as human shields. Say we-" a chirp from the proximity alert cuts him off.`
```

				goto showtime

			label plan
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`

			label showtime

> 			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`

Author: ```suggestion
			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
			`	"Oh, I don't like the look of this," says Bunker.`
			choice
				`	"Keep filming."`
				`	"Call back the drones immediately."`
					goto "call drones"
				`	"I think we'll have to leave the drones behind."`
					goto "no can do"
				`	"Did he see us?"`
					goto "see us"

			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
			choice
				`	"That's an order!"`
				`	"Okay. Call the drones back."`
					goto "call drones"

			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`

> 			`	You notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers. You try to pull out to space but another pair of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`

Reviewer 5: ```suggestion
			`	You barely notice the humming from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle down the shield down rapidly. Quickly pushing your thrusters to the limit, you commanded your ship upwards, trying to make it to vacuum to get out of system before another pair of Quicksilvers send another volley of fire that broke through and hit your ship's repulsors. The last thing you experience is the ship slowing to a halt, levitating for just a moment before the pilot seat went out underneath you, the hull filled with screams as the <ship> makes a one way trip towards the surface.`
```

Reviewer 5: Definitely not happy with my rewrite, but I wanted to make the death a lot more immersing.

Author: @TheMarksman-ES How about this?
```suggestion
			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle down the shield down in a split second. You try to shunt your repulsors to full blast to get into orbit before another pair of Quicksilvers send another volley of fire that broke through and hit your ship's repulsors. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```

				die

			label "see us"
			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`
			choice
				`	"Call back the drones immediately."`
					goto "call drones"
				`	"I think we have to leave the drones behind."`

			label "no can do"
			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
			choice
				`	"We're leaving. Now!"`
				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
					goto escape

			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
			choice

> 				`	(Charge him with mutiny.)`

Reviewer 5: Thinking on it, this guy isn't part of your crew. Also, you as a captain don't have legal rights to have someone charged.

Reviewer 4: I think under maritime law the captain of a ship in international waters can charge somebody. Since this ship is on a planet, though, I’d imagine local law would technically need to be involved. As for whether it is actually mutiny if the person isn’t technically part of your crew? No idea. Would it be attempted hijacking, maybe?

Reviewer 6: I think it doesn't matter if he's a member of your crew. He's required to obey the captain while aboard your vessel.

Reviewer 7: ```suggestion
				`	(Force him to stand down.)`
```
It's not exactly clear as-is that picking this option results in `first last` pulling their gun. I think this is a little more clear and resolves the question of mutiny vs. hijacking, but open to suggestions.

				die

	npc
		personality heroic nemesis vindictive launching
		government "Pirate"
		ship "Quicksilver" "Dog's Leash"
		ship "Quicksilver" "Burning Whip"
		ship "Quicksilver" "Iron Chains"
		on disable
			set "adelita: killed pirates"

	on visit
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

mission "Adelita 5"
	name "Return Adelita to <planet>"
	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"
	landing
	passengers 3
	cargo "stealth drones" 1
	"apparent payment" 200000
	source "Maker"
	destination "Moonshake"
	to offer
		has "Adelita 4: done"

	on offer
		conversation
			branch experienced
				has "adelita: killed pirates"

			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`

> 			`	"That was horrible, but we got it done. Can we leave now?" asks Ale.`

Reviewer 5: ```suggestion
			`	"That was horrible, but we got it done," Ale mumbled. "Can we leave now?" 
```

Reviewer 5: Need to be more in depth.

				launch

			label experienced

> 			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`

Reviewer 5: Ya... You don't just... tag seven words to the front of the paragraph. You need to have that statement flow in with Ale, River, and Bunker's experience.

			`	"That was horrible, but we got it done. Can we leave now?" asks Ale.`
				launch

	on visit
		dialog phrase "generic cargo and passenger on visit"

	on complete
		payment 220000

> 		log `Helped the underground journalism collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade. Had to escape pirates while doing so.`

Reviewer 5: ```suggestion
		log `Helped the underground journalism collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade while escaping from pirates.`
```

		conversation

> 			`Ale, River, and Bunker look glad to be back on <planet>. "Ugh. Being there in person was bad enough," says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you. We'll let you know when it's done. Also, we might need you for some future projects, so expect us to be in touch." The group leaves your ship.`

Reviewer 5: ```suggestion
			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumble, tapping against a stack of folders on her device. "Now we have to watch it over and over again to edit it into something presentable."`
			
			Before leaving, she hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you and you'll be one of the first to know when it's done. Also, we might be in need of you for some future projects, so expect us to be in touch."`
```

Reviewer 5: The original had a disconnect in emotion. Ale is "telling," or "grumbling" that they had to rewatch the footage, yet somehow, she was somewhat chirpy immediately as she hands the chips.

Reviewer 4: ‘Grumbles’, and is there an extra blank line now? 7167 seems to only be a blank line, and 7168 seems to start with a blank line too. Of course that could just be my phone formatting it weirdly.

		not attributes "textiles"
		not attributes "research"
		not attributes "factory"
	destination "Moonshake"
	cargo "exhibition installations" 5
	to offer
		random < 80
		has "Expedition to Hope 2: done"
		"combat rating" > 40

	on offer
		conversation
			`Walking around <origin>'s luxurious spaceport, you notice a large group of people gathered around one of the larger hangars; there are large signs that say "The Evacuation of Hope" in bold letters by it. It looks like some sort of exhibition.`
			choice
				`	(Check it out.)`
				`	(Not right now.)`
					defer

			`	You approach some of the people at the end of the line and ask what's going on. "It's Adelita's exhibition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their exhibitions are always free, so this is a great chance to see it if you haven't already."`
			choice
				`	"Sure, why not."`
				`	"Maybe some other time."`
					defer

			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`

> 			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here, on a Paradise Planet.`

Author: For being poor and then getting this treatment from the Navy that the Paradise Worlds paid for, I guess. The Paradise Worlds are supposed to be a bit irrational.

> 			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here, on a Paradise Planet.`

Reviewer 4: Fair enough.

I’ve got this image of a rally with signs saying things like ‘death to the poor!’ and ‘no bucks? You suck!’ except the rich folks were too busy getting their nails done to turn up so they sent their people to protest in their place. So it’s just a bunch of menial workers looking bored, unironically holding signs denouncing themselves.

> 			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here, on a Paradise Planet.`

Reviewer: ```suggestion
			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews, and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here, on a Paradise Planet.`
```

			`	There are many, many photos, but one catches your eye. It's of a woman standing in line to board a ship, a baby held at her hip and a look of sad determination on her face. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background.`

> 			`	You head towards the exit, but as the automatic doors open, an arctic blast rushes at you from every direction. Instead of the spaceport's open apron, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries gradually being covered by ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`

Reviewer: ```suggestion
			`	You head towards the exit, but as the automatic doors open, an arctic blast rushes at you from every direction. Instead of the spaceport's open apron, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```
Also maybe break that last sentence `But what stands out most is the massive volcano erupting in the background.` out into a new line?

Author: Hmm, would it be too dramatic for it to be out on another line?

It is a massive volcano...

			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. So what do you think?"`
			choice
				`	"Impressive."`
				`	"It's alright."`

			`	"Well, it was just as I imagined it," the woman replies. "Oh! I have an exam to cram for. Enjoy the exhibition, I think it closes soon." She heads for the exit.`
			choice
				`	(Continue wandering around the exhibit.)`
				`	(Go to the exit.)`
					goto exit

			branch katya
				has "FW Katya 4: done"

> 			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. In the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`

Reviewer 4: ```suggestion
			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. From the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`
```
I think this is just personal preference but I feel like it reads better this way.

				goto "move on"

			label katya

> 			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. In the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`

Reviewer 4: ```suggestion
			`	Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. From the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up.`
```

			label "move on"

> 			`	As you move on, you're startled to find yourself face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Suddenly, the entire projection disappears in a disorienting flash, leaving you in a sectioned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`

Reviewer 4: ```suggestion
			`	As you move on, you're startled to find yourself face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`
```

	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`

	on complete
		payment 80000
		event "adelita 2 offer expired" 1
		dialog
			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of crew walks up to you and hands you a note. It reads:`
			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. - Adelita Zero"`

event "adelita 2 offer expired"

mission "Adelita 2"
	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"
	name "Film Syndicate sweatshop on <planet>"
	source "Moonshake"
	destination "Maker"
	passengers 3
	"apparent payment" 200000
	cargo "stealth drones" 1
	to offer
		has "Adelita 1: done"
		not "adelita 2 offer expired"

	on offer
		conversation

> 			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen, you're able to get to the Common Grounds Cafe. The cafe has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat. Before your meal arrives, a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`

Reviewer 4: ```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat. Before your meal arrives, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`
```

			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`
			choice
				`	"I found it quite moving."`
				`	"It was OK."`
					goto wasok
				`	"It was too political for me."`
					goto political
			`	"I'm glad you liked it," she says, smiling.`
				goto choice

			label wasok
			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says, smiling.`
				goto choice

			label political

> 			`	"Oh. Unfortunately, for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`

Reviewer 4: ```suggestion
			`	"Oh. Unfortunately, for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```

				decline

			label choice
			`	"Mariel said you saw me take the photos of the woman and her baby," she says. "I really like that moment, but not many notice it." She leans in closer. "We need a captain to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows. Oh, and we have to leave today. It will pay, of course. <payment>. So what do you say? Are you in or out?" As she finishes, your order is served while she drinks her smoothie.`
			choice
				`	"In."`
				`	"Out."`
					goto out

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop. Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories, you will be going with an associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`

Reviewer 4: ```suggestion
			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop. Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories, you will be going with an associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
```

			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`
			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`

> 			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`

Reviewer: ```suggestion
			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker.`
```

			choice
				`	"I am Captain <first> <last>. Well met."`
				`	"Hi, I'm <first>."`
					goto loadup
			`	They all frown at your formality.`

			label loadup
			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
				accept

			label out
			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`
				decline

	on visit
		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`

mission "Adelita 3"
	landing
	name "Film slave transaction"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
	source "Maker"
	destination "Hippocrates"
	"apparent payment" 200000
	infiltrating
	to offer
		has "Adelita 2: done"
	passengers 1
	cargo "stealth drones" 1
	on offer
		conversation

> 			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take off for <planet>.`

Reviewer 4: ```suggestion
			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take-off for <planet>.`
```

Author: I'm not sure about this one. Maybe the dash would be needed if it were `prepare for take-off to <planet>`?

Reviewer 4: I don’t think that changes the meaning, does it? In either case it’s a synonym for ‘launch’, unless I’m reading it wrong.

Reviewer 5: If you have issues with it, use "Depart."

Author: ```suggestion
			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to depart for <planet>.`
```

				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

mission "Adelita 4"
	name "Slave trade"
	description "Pick up the team on <destination>. (You will be paid <payment> after you complete the entire job.)"
	source "Hippocrates"
	destination "Maker"
	landing
	infiltrating
	"apparent payment" 200000
	to offer
		has "Adelita 3: done"
	passengers 1
	cargo "stealth drones" 1

	on offer
		conversation
			branch "player has a fleet"

> 				"ships: Space Liner" + "ships: Transport" + "ships: Light Freighter" + "ships: Heavy Freighter" + "ships: Utility" + "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship" + "ships: Superheavy" > 1

Reviewer 3: ```suggestion
				"total ships" - ( "ships: Fighter" + "Ships: Drone" ) > 1
```

Author: If you had 3 ships and 2 fighters in your fleet, wouldn't it show the non-fleet message? Or is that acceptable to have to simplify this line?

Reviewer 3: Fighters count towards the total ships count.

Author: Yeah, I'm just wondering if you could have a fleet of something like an Aerie, two Daggers, and two Ravens, the Daggers would cancel out the Ravens in the calculation (even though they're still in your fleet as you land) so the altered text about your fleet being in orbit wouldn't show?

Whereas if you just didn't count them there'd be no problem.

Reviewer 3: If you had an Aerie, two Daggers and two Ravens, this condition would be calculated as follows:
`"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1`
`5 - ( 2 + 0 ) > 1`
`5 - 2 > 1`
`3 > 1`
and the branch will trigger to show the fleet text. The purpose of this check is to stop a solo carrier from being considered as a fleet, and having the fighters be left in orbit when they should be with the ship.

			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
				goto "on site"

			label "player has a fleet"

> 			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

Reviewer: ```suggestion
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```
Laying low in the canyon covers that, I'd think :)

> 			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation; luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

Reviewer 3: ```suggestion
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

			label "on site"
			`	Once the drones are on site, you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`
			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

			`	"And you know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`
				goto showtime

			label plan

> 			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`

Author: ```suggestion
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" a chirp from the proximity alert cuts him off.`
```

Reviewer 5: "A chirp," actually. :p

			label showtime

> 			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship in pirate colors lands near the security detail, probably deafening them with the sound of their repulsors. Accompanying it is a small flotilla of Quicksilver escorts. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; the detail hit, shove, and shout at anyone that moves too slowly for their liking.`

Reviewer 4: ```suggestion
			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

> 			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky, as if he sees something. He motions, and the Quicksilvers fire up their engines.`

Reviewer 4: ```suggestion
			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
```

	minor
	cargo "exhibition installations" 5
	source
		attributes "paradise"
		attributes "rich"
	destination "Moonshake"
	to offer
		random < 80
		has "Expedition to Hope 2: done"
		"combat rating" > 40

	on offer
		conversation
			`Walking on the tarmac of <origin>'s luxurious spaceport, you notice a significant group of people gathered around one of the larger hangars. From the signs nearby that display "The Evacuation of Hope" in bold letters, it looks like some sort of exhibition.`
			choice
				`	(Check it out.)`
				`	(Not right now.)`
					defer

			`	You approach some of the people at the end of the line and ask what's going on. "It's Adelita's exhibition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their exhibitions are always free, so this is a great chance to see it if you haven't already."`
			choice
				`	"Sure, why not."`
				`	"Maybe some other time."`
					defer

			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`

> 			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews, and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here on a Paradise Planet.`

Author: It's an irrational protest. Nevertheless, it is explained in the FW campaign:
> "Here's the best part," says Eyes. "The Navy operates with a certain amount of autonomy. Turns out when they found out about the disaster, they mobilized without even waiting to clear it with Parliament. Which was technically within their mandate, this being a natural disaster. But the Navy is funded by taxes, and the Paradise Planets pay ten times as much in taxes as the Dirt Belt."

> "The Paradise Planets viewed the Navy as their own private security force," says Katya, "so they got really mad when half the Navy suddenly vanished to assist some dirty refugees. Because those admirals gave the order to assist us, millions of lives were saved. And then Parliament fired them."

			`	There are many, many photos, but one catches your eye. It's of a woman standing in line to board a ship, a baby held at her hip and a look of sad determination on her face. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background.`

> 			`	You head towards the exit, but as the automatic doors open, an arctic blast rushes at you from every direction. Instead of the spaceport's open apron, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`

Author: It's another exhibit, part of the same exhibition; that part of the hangar was transformed to appear like Hope.

In some buildings there are heaters that warm you as you walk through the door, from above you. I imagined there was just a very effective cooling system. Might make more sense to just be in front of you though.

			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. So what do you think?"`
			choice
				`	"Impressive."`
				`	"It's alright."`

			`	"Well, it was just as I imagined it," the woman replies. "Oh! I have an exam to cram for. Enjoy the exhibition, I think it closes soon." She heads for the exit.`
			choice
				`	(Continue wandering around the exhibit.)`
				`	(Go to the exit.)`
					goto exit

			branch katya
				has "FW Katya 4: done"

> 			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. From the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`

Author: ```suggestion
			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, resolute. From the corner of your eye, you notice that a Navy officer slips in the snow, and as you look around, a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`
```

				goto "move on"

			label katya

> 			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships; it strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, distraught, frightened, resolute. From the corner of your eye you notice that a Navy officer slips in the snow, and as you look around a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`

Author: ```suggestion
			`	Inside houses you see families frantically gathering their belongings and running for the Navy ships. You walk down one of the lines of refugees, observing each person - their faces harried, frightened, resolute. From the corner of your eye, you notice that a Navy officer slips in the snow, and as you look around, a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`
```

			label "move on"

> 			`	As you move on, you're startled to find yourself face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`

Author: ```suggestion
			`	As you move on, you're startled to find yourself face-to-face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question, and begins to take photos.`
			`	Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`
```

			`	"Captain," she says, her eyes lighting up. "Well, as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`
			choice
				`	"I would love to."`
				`	"Sorry, I have more pressing matters at hand."`
					goto decline

			`	"Great!" Mariel says. "Bring your ship to this hangar, and I will have our crew get everything ready to load."`
			`	After a couple hours, the whole exhibition is packed up and ready for transport to <planet>.`
				accept

			label decline
			`	"Oh. I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call." Mariel glances at her watch. "Well anyway, the exhibit is closed. I'm going to have to ask you to leave."`
				decline

			label exit
			`	Following where the woman went, you find the true exit of the hangar and go back out onto the familiar spaceport tarmac.`
				decline

	on accept

> 		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`

Author: I don't think it works, unfortunately. Substitutions are not used anywhere else in logs as far as I can see.

Reviewer 6: Deleted my incorrect suggestion, but it looks like it left your reply orphaned

	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`

	on complete
		payment 80000
		event "adelita 2 offer expired" 1

> 		dialog
> 			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of the crew walks up to you and hands you a note. It reads:`
> 			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed. - Adelita Zero"`

Author: ```suggestion
		dialog
			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of the crew walks up to you and hands you a credit chip for <payment> and a note. It reads:`
			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed. - Adelita Zero"`
```
Assume the credit chip is one that only the player can cash. Then it makes sense that a rando gives you 80k credits.

It's not established whether to use a dialog box or a conversation to end a mission, existing missions do both, but I thought dialog was more appropriate for a note.

> 			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed. - Adelita Zero"`

Reviewer 6: Hmm. I don't know. It seems kind of large/cramped to me for a dialog box, but it's ultimately up to you. Just a soft suggestion.

<img width="854" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/80cd9e70-6807-40cd-9484-b227a714dc16">

The part about the payment that you're adding looks good, by the way.

			label leave
			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker.`
			choice
				`	"I am Captain <first> <last>. Well met."`
				`	"Hi, I'm <first>."`
					goto loadup
			`	They all frown at your formality. "Well, let's load up," says Bunker.`
				accept

			label loadup
			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
				accept

			label out
			`	"I'm sorry to hear that... I suppose it just wasn't meant to be." She gulps down her smoothie at lightning speed before leaving just as quickly.`
				decline

	on visit
		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`

> 

Author: ```suggestion

```

mission "Adelita 3"
	name "Film slave transaction"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
	landing
	cargo "stealth drones" 1
	passengers 1
	infiltrating
	"apparent payment" 200000
	source "Maker"
	destination "Hippocrates"
	to offer
		has "Adelita 2: done"

	on offer
		conversation

> 			`You drop off Ale and River at the spaceport, watching as they try not to seem either visibly embarrassed to be in their security uniforms, or uncomfortable with their disguises, even though they're now virtually indistinguishable as Syndicate personnel; Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`

Reviewer 6: ```suggestion
			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel; Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
```

				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

> 

Author: ```suggestion

```

			label "on site"
			`	Once the drones are on site, you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`
			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
				goto showtime

			label plan
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`

			label showtime

> 			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`

Reviewer 6: ```suggestion
			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

					goto "no can do"
				`	"Did he see us?"`
					goto "see us"

			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
			choice
				`	"That's an order!"`
				`	"Okay. Call the drones back."`
					goto "call drones"

			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`
			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle the shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers send another volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`

> 				die

Reviewer 6: I'm not so sure this should result in insta-death. The thought that a couple of Quicksilvers armed with particle cannons could tear through a capital ship's shield before you have time to react or get away... it beggars belief. I think the player should be launched into orbit and have to fight for their life.

If you really, _really_ want to force insta-death, then you should change the text to remove references of going through the ship's shields. The player must have had their shields down to avoid interfering with the drone signals, or the ship was intentionally on low power mode to avoid attracting attention. Ideally, in fact, I would have the player in a vulnerable situation. Perhaps they stuck their head out of the cockpit or something like that. Otherwise, you're going to end up in an unrealistic situation where someone is scoffing at the fact that three Quicksilvers took down their Hurricane, for example.

Author: I will think about rewriting it.

Author: Ok, what I could do is let the player run away, but decline this mission (`flee`). This allows you to complete this mission chain, but you won't be able to continue doing Adelita missions down the line since you weren't able to recover the footage. Separate missions will take care of the different dialogue that will arise due to this, and you won't recieve as much money as before. After all, the player can't always be forced to pick the right option, or otherwise we're dictating to them what we want the story to go in a supposedly multi-choice game. I could even let the player re-establish themselves with Adelita in a follow-up mission, though I'd rather get the rest of #5112 done first.

			label "see us"
			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`
			choice
				`	"Call back the drones immediately."`
					goto "call drones"
				`	"I think we have to leave the drones behind."`

			label "no can do"

> 			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`

Reviewer 6: ```suggestion
			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we'll lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
```

			choice
				`	"We're leaving. Now!"`
				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
					goto escape

> 			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`

Reviewer 6: ```suggestion
			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down, and we're out of here.`
```

Author: Maybe would be better without the comma since it's meant to have a slightly rapid pace.

			choice
				`	(Charge him with mutiny.)`
					goto blaster
				`	(Let him go to the cargo bay and lock down the drones.)`
					goto escape

			label "call drones"
			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`

			label escape
			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`

> 			`	The moment you pull out of the canyon, pirate Quicksilvers appear in the distance from all directions. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`

Reviewer 6: ```suggestion
			`	The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
```

				launch

			label blaster
			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, whittling the <ship>'s shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`

> 				die

Reviewer 6: Same comment about insta-death as above

	npc
		personality heroic nemesis vindictive launching
		government "Pirate"
		ship "Quicksilver" "Dog's Leash"
		ship "Quicksilver" "Burning Whip"
		ship "Quicksilver" "Iron Chains"
		on disable
			set "adelita: killed pirates"

	on visit
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

> 

Author: ```suggestion

```

mission "Adelita 5"
	name "Return Adelita to <planet>"
	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"
	landing
	passengers 3
	cargo "stealth drones" 1
	"apparent payment" 200000
	source "Maker"
	destination "Moonshake"
	to offer
		has "Adelita 4: done"

	on offer
		conversation
			branch "killed pirates"
				has "adelita: killed pirates"

> 			`You managed to evade the Quicksilvers and arrive on <origin> alive. As soon as you land Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`

Reviewer 6: ```suggestion
			`You managed to evade the Quicksilvers and arrive on <origin> alive. As soon as you land, Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
```

			`	"That was horrible, but we got it done," Ale mumbles. "Can we leave now?"`
				launch

			label "killed pirates"

> 			`You managed to neutralize the Quicksilvers and arrive on <origin> alive. As soon as you land Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, all of them are eager to get back and process what they filmed - both emotionally and literally.`

Reviewer 6: ```suggestion
			`You managed to neutralize the Quicksilvers and arrive on <origin> alive. As soon as you land, Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, all of them are eager to get back and process what they filmed - both emotionally and literally.`
```

> 			`You managed to neutralize the Quicksilvers and arrive on <origin> alive. As soon as you land Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, all of them are eager to get back and process what they filmed - both emotionally and literally.`

Reviewer 6: If this is the only reason for the `killed pirates` flag, then you could eliminate all of that and just use a different wording:

`You managed to deal with the Quicksilvers and...`

Author: It would be simpler.

			`	The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
				launch

			label blaster
			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, whittling the <ship>'s shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
				die

	npc
		personality heroic nemesis vindictive launching

> 		government "Pirate"

Author: Do we want Syndicate ships to fight these pirates that are helping them? "Bounty Hunter" doesn't make sense, so maybe I'll make a new government called "Pirate (Core Slavers)" who are friendly with the Syndicate.

			action
				payment 120000
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`

mission "Adelita 1"
	name "Exhibition installations to <planet>"
	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
	minor
	cargo "exhibition installations" 5
	source "Follower"
	destination "Moonshake"
	to offer
		or
			random < 10
			and
				random < 60
				has "Expedition to Hope 2: done"
		"combat rating" > 40

	on offer
		conversation
			branch "has visited"
				has "visited planet: Hope"

> 			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you vaguely remember it as being a planet in a system southwest of New Boston. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`

Reviewer 8: I think its explained in FW what happened (by Katya iirc?) and this mission doesnt seem to check for that

Author: Yeah, I could update it to add another branch if you've done Katya's missions but still haven't been to Hope.

> 			branch "has visited"
> 				has "visited planet: Hope"
> 			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you vaguely remember it as being a planet in a system southwest of New Boston. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
> 				goto directions
> 
> 			label "has visited"
> 			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you remember having been there once before on an excursion. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`

Author: ```suggestion
			branch "know about disaster"
				has "FW Katya 4: done"
			branch "has visited"
				has "visited planet: Hope"
			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you vaguely remember it as being a planet in a system southwest of New Boston. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
				goto directions

			label "has visited"
			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you remember having been there once before on an excursion. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
				goto directions

			label "know about disaster"
			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you remember talking about it with Katya and Ijes while you were investigating the source of the nuclear bombings. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
```

			label directions
			`	At the bottom of each sign are the directions to get to the exhibition. It appears to be located in a warehouse yard nearby, where the spaceport district borders the city that surrounds it.`
			choice
				`	(Check it out.)`
				`	(Not interested.)`
					decline
			`	Following the signage you notice a significant group of people gathered in front in front of a large warehouse that stands out amongst the rest of the warehouses in the yard. As you wait in the queue a young woman lines up behind you with a readily apparent exuberance and asks you, "I'd like to know, out of interest, how much you know about Hope and its history?"`
			choice
				`	"Not much at all."`
				`	"I come from the Dirt Belt, but I really don't know too much about what happened on Hope."`
				`	"I've been there myself and know what happened, but other than that I don't know."`
					goto "been to hope"
					to display
						has "visited planet: Hope"

> 						or

Author: If you've been to Hope but not done either of those two missions, you know someone who's been affected and have been told why.

> 						or

Reviewer 6: Are the options not intended to be mutually exclusive? I am getting both of them when I have done only one of the missions.
<img width="577" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/b5a4f35f-d6e2-44e3-8385-7d81d2db4f94">
If you want them to be mutually exclusive (either you know someone or you don't), then you need an `and` instead of an `or`.

				goto questions

			label "been to hope"
			`	"Oh, wow! It must have been a harrowing experience. Adelita does a powerful job of giving you the whole story, though, and I bet you'll still learn a lot. You might even feel as if you've been transported back there yourself," she says with a mischievous grin. "You know, this was their first work that really got them famous in the interplanetary scene, but they've been forced to close it and today's their final day. I had to see it one more time before they went."`

			label questions
			choice
				`	"Who's Adelita?"`
				`	"Why did they have to close it?"`
					goto signage

			`	"They're an independent journalistic collective whose goals are to bring the injustices of human civilization to light. No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their use of mixed media, their eye for story telling... I just love it. I know you're going to love it too," she says.`
				goto exhibition

			label signage

> 			`	"Well, this exhibition used to be permanent, and it garnered a lot of attention, but the planetary government was pressured by the Paradise Planets to force Adelita to leave, because it gave such a candid view of what they did to those poor people. Personally, I think what Parliament and the Paradise Planets have been doing to the people of Hope and their advocates like Adelita has just been abhorrent and wrong."`

Reviewer 7: ```suggestion
			`	"Well, this exhibition used to be permanent, but the planetary government was pressured by the Paradise Planets to force Adelita to leave. What the Navy and the Paradise Planets did to the people of Hope was abhorrent. Now Adelita's bringing light to it, and they're being silenced. It's wrong."`
```
Same as before; I just think this sounds more like how someone would talk. Also, it's mentioned later that this was more of a Navy thing than from Parliament. Open to suggestions!

Author: It was the Navy that acted on their own initiative to save the people of Hope, and Parliament that punished them for doing so.

> "The eruption was on the other continent," she explains, "so no one was killed in the blast. It was a slow-moving disaster. We had six months or so to move two million settlers off-world before the temperatures would drop too low to be livable. More than half of the Navy was mobilized to ferry our people to other planets."
> "Here's the best part," says Eyes. "The Navy operates with a certain amount of autonomy. Turns out when they found out about the disaster, they mobilized without even waiting to clear it with Parliament. Which was technically within their mandate, this being a natural disaster. But the Navy is funded by taxes, and the Paradise Planets pay ten times as much in taxes as the Dirt Belt."
> "The Paradise Planets viewed the Navy as their own private security force," says Katya, "so they got really mad when half the Navy suddenly vanished to assist some dirty refugees. Because those admirals gave the order to assist us, millions of lives were saved. And then Parliament fired them."

Author: ```suggestion
			`	"Well, this exhibition used to be permanent, but the planetary government was pressured by the Paradise Planets to force Adelita to leave. What Parliament and the Paradise Planets did to the people of Hope was abhorrent. Now Adelita's bringing light to it, and they're being silenced. It's wrong."`
```

			label exhibition
			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews, and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them are quite crude and tasteless. It's a strong rebuke to see those displayed here, so close to the Paradise Planets.`

> 			`	There are many, many photos, but one in particular catches your eye. It's of a woman standing in line to board a ship, a baby held at her hip and a look of sadness and determination on her face. The mother and child's cheeks, rosy and chapped, stand out in contrast with the cold blues in the background.`

Reviewer 7: When I reviewed this originally, this reminded me of the [Afghan Girl pic](https://en.wikipedia.org/wiki/Afghan_Girl). Can't remember if that was intended or not, but it's quite fitting, I think.

> 			`	You head towards the exit, but as the automatic doors open, an arctic blast pummels into you from out of nowhere. Instead of <origin>'s spaceport, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`

Reviewer 7: ```suggestion
			`	You head toward the exit, but as the automatic doors open, an arctic blast pummels into you from out of nowhere. Instead of <origin>'s spaceport, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```

> 			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. So what do you think?"`

Reviewer 7: ```suggestion
			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. What do you think?"`
```

			choice
				`	"Impressive."`
				`	"It's alright."`

> 			`	"I've been here ten times and I always manage to be awestruck," the woman replies. "Enjoy the exhibition; I think it will close soon, and for the last time." She heads for the exit.`

Reviewer 7: ```suggestion
			`	"I've been here ten times and I always manage to be awestruck," the woman replies. "Enjoy the exhibition. I think it's closing soon, and for the last time." She heads for the exit.`
```

				goto "move on"

			label katya
			`	Inside houses, you see families frantically gathering their belongings and running for the Navy ships; it strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person - their faces harried, frightened, resolute. From the corner of your eye, you notice that a Navy officer slips in the snow, and as you look around, a young man helps him up. Wandering around the exhibit gives you a sense of the desperation of the people as they evacuate, though it's a bit disconcerting when the occasional hologram walks through you.`

			label "move on"
			`	As you move on, you're startled to find yourself face-to-face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question, and begins to take photos.`

> 			`	Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk.`

Reviewer 7: ```suggestion
			`	Suddenly, the entire projection disappears in a disorienting flash, leaving you in a cordoned-off section of the hangar bay. The exhibit is nearly empty as a woman walks up to you. "One of the authors loves that moment too. That's probably why she fought so hard to have that part left in." She mutters something about a "security risk" under her breath, and then turns her attention back to you.`
```
I vaguely recall that this context of security is important to the story, but it seems like a really weird thing for Mariel to just mention out of the blue to a stranger. Thoughts?

Author: That's fair enough, since Adelita is so protective of its privacy and Alejandra keeps this up all the way to when you agree to help her.

			`	"Oh, how rude of me," she says, offering her hand. "My name is Mariel. I organize these exhibitions and act as a spokesperson for Adelita. And you are?"`

> 			choice
> 				`	"Captain <last>."`

Reviewer 7: ```suggestion
			choice
				`	"Captain <last>."`
				`	"<first> <last>, Captain of the <ship>."`
```

> 			choice

Reviewer 7: Thoughts on adding another choice? Just another way for the player to introduce themselves? Would still need to specify  or imply that they're a ship captain.

Author: It could be worth it, at least to avoid 'railroading' the story with one choice. Extra dialogue / choice nodes can specify they're a captain.

> 			`	"Captain," she says, her eyes lighting up. "Well, as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`

Reviewer 7: ```suggestion
			`	"Captain," she says, her eyes lighting up. "Well, as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I'll pay you <payment> to take his contract instead given the short notice. What do you say?"`
```

			choice
				`	"I would love to."`

> 				`	"Sorry, I have more pressing matters at hand."`

Reviewer 7: ```suggestion
				`	"Sorry, but I have more pressing matters at hand."`
```

> 			choice
> 				`	"I would love to."`
> 				`	"Sorry, but I have more pressing matters at hand."`
> 					goto decline
> 
> 			`	"Great!" Mariel says. "Bring your ship to this hangar, and I will have our crew get everything ready to load."`
> 			`	After a couple hours, the whole exhibition is packed up and ready for transport to <planet>.`
> 				accept
> 
> 			label decline
> 			`	"Oh. I thought that would be a pretty good deal for you, especially given the chance to meet the artists. But it's your call." Mariel glances at her watch. "Well, anyway, the exhibition has closed. I'm going to have to ask you to leave."`
> 				decline

Author: ```suggestion
			choice
				`	"I would love to."`
					goto accept
				`	"I've got more pressing matters at hand."`

			`	"Oh. I thought that would be a pretty good deal for you, especially given the chance to meet the artists. But it's your call."`
			choice
				`	"Okay, I'll do it."`
				`	"I'm sorry, I can't take the job."`
					goto decline

			label accept
			`	"Great!" Mariel says. "Bring your ship to this hangar, and I will have our crew get everything ready to load."`
			`	After a couple hours, the whole exhibition is packed up and ready for transport to <planet>.`
				accept

			label decline
			`	"That's unfortunate. I'll have to try my luck with another captain, then." Mariel glances at her watch. "Well, anyway, the exhibition has closed. I'm going to have to ask you to leave."`
				decline
```

> 			`	"Oh. I'm surprised you would turn down so much money for such little work as well as a chance to meet the artists. But it's your call." Mariel glances at her watch. "Well, anyway, the exhibition has closed. I'm going to have to ask you to leave."`

Reviewer 7: ```suggestion
			`	"Oh. I thought that would be a pretty good deal for you, especially given the chance to meet the artists. But it's your call." Mariel glances at her watch. "Well, anyway, the exhibition has closed. I'm going to have to ask you to leave."`
```
The way this is worded originally seems like Mariel's almost trying to talk `first last` into changing their mind. Maybe it makes sense to have another choice below where they get another chance to accept? If so, put the refusal as the first choice and acceptance as the second.

			label exit
			`	Following where the woman went, you find the true exit of the warehouse and go back out onto the more familiar grounds of <origin>'s spaceport.`
				decline

	on accept
		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`

	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`

	on complete
		payment 80000
		event "adelita 2 offer expired" 1
		conversation
			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of the crew walks up to you and hands you a credit chip for <payment> and a note. It reads:`
			``
			`Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed.`
			`	Adelita Zero`

event "adelita 2 offer expired"

mission "Adelita 2"
	name "Film Syndicate sweatshop on <planet>"

> 	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"

Reviewer 7: ```suggestion
	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. You will be paid <payment> after you complete the entire job."
```

	cargo "stealth drones" 1
	passengers 3
	"apparent payment" 200000
	source "Moonshake"
	destination "Maker"
	to offer
		has "Adelita 1: done"
		not "adelita 2 offer expired"

	on offer
		conversation

> 			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`

Reviewer 6: ```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen, you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
```

Author: One too many commas for me, I think it breaks the flow.

Reviewer 7: ```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>. After finding an information screen, you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
```
How about this? I like the pause, but I agree that the sentence feels run-on-ish with just a comma thrown in. I personally think giving the longer pause with the period sounds better.

> 			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of nor can even pronounce.`

Reviewer 7: ```suggestion
			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have neither heard of nor can pronounce.`
```

> 			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`

Reviewer 6: ```suggestion
			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of nor can even pronounce.`
```

> 			`	Before your meal arrives, however, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`

Reviewer 6: There is some risk here, because you are dealing with fixed dates. If the evacuation of Hope took place in 2994 and the player is doing this mission in, say, 3030, then the woman cannot be in her thirties. Even in the best case scenario, if the woman is 39, she would have only been 19 when taking the photos, since we're starting the game in 3014.

I'm not quite sure how to fix this without just deleting the bit about her being in her late thirties. Maybe late forties would give some more wiggle room?

			`	"Sorry for the cloak and dagger bit," the woman says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`
			choice
				`	"I found it quite moving."`

> 				`	"It was OK."`

Reviewer 7: ```suggestion
				`	"It was okay."`
```

				decline

			label choice
			`	"I heard from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She takes a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, and then pick up the team and return here with everyone safe and sound. I can't tell you much more than that until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`
			`	"Oh, and we have to leave today. It will pay, of course. <payment>. So, what do you say? Are you in or out?" When she finishes speaking, your order is served while she drinks her smoothie.`
			choice
				`	"In."`
				`	"Out."`
					goto out

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop.`

Reviewer 7: ```suggestion
			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet>, where we'll to film the conditions in a secret Syndicate sweatshop.`
```

> 			`	"Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking, we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' is getting the slaves from the pirates on Covert, and we also know where they are doing the transaction.`

Reviewer 7: ```suggestion
			`	"Thing is, these aren't just poorly paid factory workers. They're slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking, we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' is getting the slaves from the pirates on Covert, and we also know where they're doing the transaction.`
```

> 			`	"So while we are getting the footage we need in the factories, you will be going with my associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`

Reviewer 7: ```suggestion
			`	"So while we gey the footage we need in the factories, you'll going with my associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
```

Author: ```suggestion
			`	"So while we get the footage we need in the factories, you'll be going with my associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
```

			choice
				`	"Hold on, pirates?"`
				`	"Won't this get me in trouble with the Syndicate?"`
					goto syndicate

> 			`	"Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying. You won't be anywhere near them, but you might encounter some lone pirates on the way, so you'll need to be quick."`

Reviewer 7: ```suggestion
			`	"Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying. You won't be anywhere near them. You might encounter some lone pirates on the way though, so you'll need to be quick."`
```

				goto leave

			label syndicate

> 			`	"Well, I did say we had enemies in high places, didn't I? But don't worry, Bunker's a pro. We have an array of long-range stealth drones that do the spying. You won't be anywhere near them, though pirates do frequent the system, so you'll need to be quick."`

Reviewer 6: ```suggestion
			`	"Well, I did say we had enemies in high places, didn't I? But don't worry. Bunker's a pro. We have an array of long-range stealth drones that do the spying. You won't be anywhere near them, though pirates do frequent the system, so you'll need to be quick."`
```

			label leave
			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
			`	You go back to the <ship>'s landing pad, and sure enough, there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker, and they enthusiastically stow their equipment away without a hitch.`
			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
				accept

			label out
			`	"I'm sorry to hear that... I suppose it just wasn't meant to be." She gulps down her smoothie at lightning speed before leaving just as quickly.`
				decline

	on visit
		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`

mission "Adelita 3"
	name "Film slave transaction"

> 	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"

Reviewer 7: ```suggestion
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. You will be paid <payment> after you complete the entire job."
```

	landing
	cargo "stealth drones" 1
	passengers 1
	infiltrating
	"apparent payment" 200000
	source "Maker"
	destination "Hippocrates"
	to offer
		has "Adelita 2: done"

	on offer
		conversation

> 			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel; Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`

Reviewer 7: ```suggestion
			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel. Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
```

				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

mission "Adelita 4"
	name "Take the film to <planet>"

> 	description "Take the film to <destination>, where the rest of the Adelita group remained behind. (You will be paid <payment> after you complete the entire job.)"

Reviewer 7: ```suggestion
	description "Take the film to <destination>, where the rest of the Adelita group remained behind. You will be paid <payment> after you complete the entire job."
```

	landing
	passengers 1
	cargo "stealth drones" 1
	infiltrating
	"apparent payment" 200000
	source "Hippocrates"
	destination "Maker"
	to offer
		has "Adelita 3: done"

	on offer
		conversation
			branch "player has a fleet"
				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1

> 			`It's not hard to find a remote canyon within range of the target area. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape. You are low enough that Bunker can release the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
> 				goto "on site"
> 
> 			label "player has a fleet"
> 			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

Author: ```suggestion
			`It's not hard to find a remote canyon within range of the target area. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`
				goto release

			label "player has a fleet"
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`
			label release
			`	After settling on a secure location within the canyon, Bunker releases the drones from the open cargo bay; the drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

> 			label "on site"

Author: ```suggestion
```

> 			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
> 				goto "on site"
> 
> 			label "player has a fleet"
> 			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

Reviewer 6: ```suggestion
			`It's not hard to find a remote canyon within range of the target area. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape. You are low enough that Bunker can release the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
				goto "on site"

			label "player has a fleet"
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

Shields need to be off if we're going to be expected to believe that the Quicksilvers can kill you later.

> 			`	Once the drones are on site, you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`

Reviewer 6: ```suggestion
			`	As you watch the display, the drones arrive on site, and you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
```
This makes it more clear that you are watching the action from the drones' point of view (on the display). When I first read this, I thought you could see what was happening from the vantage point of the ship.

> 			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`

Reviewer 6: ```suggestion
			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
```

> 			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`

Reviewer 7: ```suggestion
			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now. There's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
```

Author: Not sure about this. I don't think the full stop / period emphasises how liberating the slaves won't do much, compared to the semicolon. It's also two shortish sentences next to each other which to me is a little jarring.

Reviewer 7: Sure, makes sense. I'm good with keeping this as-is then. Feel free to marked resolved once you see this!

			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
				goto showtime

			label plan

> 			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`

Reviewer 7: ```suggestion
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
```

			label showtime
			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
			`	"Oh, I don't like the look of this," says Bunker.`

> 			choice

Reviewer 7: Let's restructure this a little bit. I'd prefer if the options leading to `die` aren't the first choice out of all the options. Up to you what order you'd like to put them in outside of that.

				`	"Keep filming."`

> 				`	"Call back the drones immediately."`

Reviewer 7: ```suggestion
				`	"Call back the drones now."`
```

					goto "call drones"
				`	"I think we'll have to leave the drones behind."`
					goto "no can do"
				`	"Did he see us?"`
					goto "see us"

> 			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`

Reviewer 6: ```suggestion
			`	"I don't know. I think we oughta get going," says Bunker, grimacing.`
```
Don't need an apostrophe for `oughta`

> 				`	"Keep filming."`
> 				`	"Call back the drones now."`
> 					goto "call drones"
> 				`	"I think we'll have to leave the drones behind."`
> 					goto "no can do"
> 				`	"Did he see us?"`
> 					goto "see us"
> 
> 			`	"I don't know. I think we oughta get going," says Bunker, grimacing.`
> 			choice
> 				`	"That's an order!"`
> 				`	"Okay. Call the drones back."`
> 					goto "call drones"

Author: ```suggestion
				`	"Did he see us?"`
					goto "see us"
				`	"Call back the drones now."`
					goto "call drones"
				`	"I think we'll have to leave the drones behind."`
				`	"Keep filming."`
					goto "no can do"

			`	"I don't know. I think we oughta get going," says Bunker, grimacing.`
			choice
				`	"Okay. Call the drones back."`
					goto "call drones"
				`	"That's an order!"`
```

			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`

> 			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle the shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers send another volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`

Reviewer 6: ```suggestion
			`	You hear the sudden hum of thrusters from outside, just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly out of nowhere as it tears through your hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers sends a volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```

The repulsors were deafening in the previous text, but now you barely notice them?
I'm also suggesting that you have the Quicksilvers tear through the hull in the death scene. It would be insane to imagine particle cannons breaking through the shield of a T2 ship in a short time.

Author: The repulsors would be deafening... if you were close by the ships. In both cases you are far away.

				die

			label "see us"

> 			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`

Reviewer 7: ```suggestion
			`	"I have no idea," says Bunker. "But it's probably time to go."`
```

			label "no can do"
			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we'll lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
			choice
				`	"We're leaving. Now!"`
				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
					goto escape

			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
			choice
				`	(Charge him with mutiny.)`
					goto blaster
				`	(Let him go to the cargo bay and lock down the drones.)`
					goto escape

			label "call drones"
			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`

			label escape
			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`

> 			`	The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`

Reviewer 6: ```suggestion
			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
```
Bringing the shields back up.

				launch

			label blaster

> 			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, whittling the <ship>'s shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`

Reviewer 6: ```suggestion
			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, tearing through the <ship>'s hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```

				die

	npc

> 		personality heroic nemesis vindictive launching

Author: ```suggestion
		personality heroic nemesis vindictive launching marked
```

		government "Pirate"
		ship "Quicksilver" "Dog's Leash"
		ship "Quicksilver" "Burning Whip"
		ship "Quicksilver" "Iron Chains"

	on visit
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

mission "Adelita 5"
	name "Return Adelita to <planet>"

> 	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"

Reviewer 7: ```suggestion
	description "Bring everyone back to <destination>. You will be paid <payment> after you complete the entire job."
```

	landing
	passengers 3
	cargo "stealth drones" 1
	"apparent payment" 200000
	source "Maker"
	destination "Moonshake"
	to offer
		has "Adelita 4: done"

	on offer
		conversation
			`You managed to deal with the Quicksilvers and arrive on <origin> alive. As soon as you land, Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, all of them are eager to get back and process what they filmed - both emotionally and literally.`
			`	"That was horrible, but we got it done," Ale mumbles. "Can we leave now?"`
				accept

	on visit
		dialog phrase "generic cargo and passenger on visit"

	on complete
		payment 220000

> 		log `Helped the underground journalism collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade while escaping from pirates.`

Reviewer 6: ```suggestion
		log `Helped the underground journalism collective "Adelita" gather video evidence of Syndicate participation in the slave trade while escaping from pirates.`
```
I assumed that Adelita Zero = Alejandra? The collective is previously only referred to as the "Adelita Collective" or "Adelita."

Author: I think there may be more context about why the "Zero" is there in the later missions, I'll check.

Author: Doesn't look like it. I think they only call themselves Adelita Zero in the note.

		conversation
			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`

> 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`

Reviewer 6: ```suggestion
			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
```

> 			`	She starts off to join River and Bunker out in the spaceport, before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

Reviewer 7: ```suggestion
			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
```

			`	You head toward the exit, but as the automatic doors open, an arctic blast pummels into you from out of nowhere. Instead of <origin>'s spaceport, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. What do you think?"`
			choice
				`	"Impressive."`

> 				`	"It's alright."`

Reviewer 9: Maybe an "isn't that a bit deceptive?" response?

Author: Will do.

> 			`	"I've been here ten times and I always manage to be awestruck," the woman replies. "Enjoy the exhibition. I think it's closing soon, and for the last time." She heads for the exit.`

Reviewer 9: > I think it's closing soon, and for the last time.

She kinda already told me that.

			`	"Thing is, these aren't just poorly paid factory workers. They're slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking, we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' is getting the slaves from the pirates on Covert, and we also know where they're doing the transaction.`
			`	"So while we get the footage we need in the factories, you'll be going with my associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
			choice
				`	"Hold on, pirates?"`
				`	"Won't this get me in trouble with the Syndicate?"`
					goto syndicate

			`	"Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying. You won't be anywhere near them. You might encounter some lone pirates on the way though, so you'll need to be quick."`
				goto leave

			label syndicate
			`	"Well, I did say we had enemies in high places, didn't I? But don't worry. Bunker's a pro. We have an array of long-range stealth drones that do the spying. You won't be anywhere near them, though pirates do frequent the system, so you'll need to be quick."`

> 			label leave

Reviewer 9: I feel like after learning more details about the mission, it'd make sense to provide a second option to say you're in or out.

			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
			`	You go back to the <ship>'s landing pad, and sure enough, there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker, and they enthusiastically stow their equipment away without a hitch.`
			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
				accept

			label out
			`	"I'm sorry to hear that... I suppose it just wasn't meant to be." She gulps down her smoothie at lightning speed before leaving just as quickly.`
				decline

	on visit

> 		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`

Reviewer 9: You could also have missing passengers.

Author: Replaced it with a generic dialog phrase.

mission "Adelita 3"

> 	name "Film slave transaction"
> 	description "Do a flyby of <destination> while Bunker spies on the slave transaction. You will be paid <payment> after you complete the entire job."
> 	landing

Reviewer 9: ```suggestion
	landing
	name "Film slave transaction"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction. You will be paid <payment> after you complete the entire job."
```

	cargo "stealth drones" 1
	passengers 1
	infiltrating
	"apparent payment" 200000
	source "Maker"
	destination "Hippocrates"
	to offer
		has "Adelita 2: done"

	on offer
		conversation
			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel. Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
				accept

	on visit

> 		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

Reviewer 9: Could also be missing the passenger here as well.

Author: Ditto.

mission "Adelita 4"

> 	name "Take the film to <planet>"
> 	description "Take the film to <destination>, where the rest of the Adelita group remained behind. You will be paid <payment> after you complete the entire job."
> 	landing

Reviewer 9: ```suggestion
	landing
	name "Take the film to <planet>"
	description "Take the film to <destination>, where the rest of the Adelita group remained behind. You will be paid <payment> after you complete the entire job."
```

	passengers 1
	cargo "stealth drones" 1
	infiltrating
	"apparent payment" 200000
	source "Hippocrates"
	destination "Maker"
	to offer
		has "Adelita 3: done"

	on offer
		conversation
			branch "player has a fleet"
				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1

> 			`It's not hard to find a remote canyon within range of the target area. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`

Reviewer 9: > Being so close to the surface of <origin> you're forced to deactivate your shields

What?

Reviewer 9: If this is to justify getting your ship blasted by a few Quicksilvers, I don't think we should go this route. People already take issue with the Pirate Troubles mission where you can die under similar circumstances. Trying to add an explanation for why that can happen doesn't really work.

Either we should somehow take into account the strength of the player's ship to determine if you die outright or if you can escape with some scratches (new mechanic?), or we need to change the circumstances of the mission such that you could still die due to the Quicksilvers strafing you. Perhaps you need to move on foot to a location to deploy the drones, and you can get hit while outside your ship.

Author: > Being so close to the surface of you're forced to deactivate your shields

There are a few instances where _parked_ ships have deactivated shields; e.g. the R.N.S. Susquehanna on Hope & the guy you meet in Pug space who tells you a captain in an Argosy blew up a Pug ship with their shields down.

However, I get your point. I'm just saying ships deactivating their shields near the surfaces of planets is 100% not unprecedented.

Author: I'll replace it with you and Bunker going on foot.

				goto release

			label "player has a fleet"
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`

			label release
			`	After settling on a secure location within the canyon, Bunker releases the drones from the open cargo bay; the drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`

> 

Reviewer 9: ```suggestion
```

			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
				goto showtime

			label plan
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`

			label showtime

> 			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`

Reviewer 9: Why name the Quicksilvers but not what ship the capital ship is?

Author: Because I didn't think about that. ~~Blame the person who originally wrote this.~~

Author: Done. I couldn't think of good names so I just picked names from Tintin.

Reviewer 9: I meant name what type of ship the capital ship is. We say that you see "a small flotilla of _Quicksilver_" and "a lone Syndicate-built _freighter_" (ends up specifically being a Bulk Freighter, but it's not like we have a lot of options for what that could be), but then this text just says "a capital ship," which could be one of many options.

Author: ```suggestion
			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a Vanguard flying pirate colors, accompanied by a small flotilla of Quicksilver escorts, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

Author: Done. I did mention later that a Vanguard was chasing you into orbit but I've also put that mention here.

			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
			`	"Oh, I don't like the look of this," says Bunker.`
			choice
				`	"Did he see us?"`
					goto "see us"
				`	"Call back the drones now."`
					goto "call drones"
				`	"I think we'll have to leave the drones behind."`
				`	"Keep filming."`

> 					goto "no can do"

Reviewer 9: I don't think this goto is supposed to be here. It sounds like the "keep filming" choice is what leads to the choice below this, but it's actually the "leave the drones behind" choice that's leading to the text below.

Author: ```suggestion
```

			label escape
			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`
			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
				launch

			label blaster
			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, tearing through the <ship>'s hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`

> 				die

Reviewer 9: I don't imagine it took you two minutes to pull out your blaster and restrain Bunker, so it seems odd to me that this choice would lead to the same outcome as deciding to stay and get more footage.

Perhaps we have an ending where you escape but don't save the drones, therefore causing different dialog in the next mission.

Author: I originally didn't consider this because I kinda wanted the player to progress... but then again, I said earlier I don't want to railroad people if they make the wrong decision. I'll make some changes soon.

(Also there's the conversation in the middle where he complains, so that's like at least 30 seconds of dialogue right????????)

	npc

> 		personality heroic nemesis vindictive launching marked

Reviewer 9: Do we really want it to be the case that the Syndicate wouldn't fire on these ships, though? That would imply that the whole of the Syndicate knows that these are pirates that they are making deals with. Would that not make it more obvious that the Syndicate is involved in the slave trade? That would also presuppose that the Syndicate actually are involved in the slave trade, rather than the possibility that the sweatshop is a single rogue element of some sort.

Author: The Syndicate is supposed to have enough plausible deniability to consciously make the decision to employ slaves – I believe that's what Lake told me back in November, anyway.

Author: Got rid of `marked` but still waiting on your opinion of the above.

Reviewer 9: While the Syndicate are undoubtedly not the most benevolent force in the galaxy, I don't think it makes sense to continually display them as malevolent at the same time, as that becomes very one-note/dimensional.

A dynamic I've described in the past is that I'd like to see the Syndicate as being portrayed as law-abiding... except for the fact that their level of control in Parliament allows them to influence the law. This means that the Syndicate's "plausible deniability" is the fact that they actually do follow the laws of the Republic, even if they are the ones shaping those laws to a major extent. Given that, the Syndicate as an organization wouldn't knowingly disregard slavery laws within their own boundaries, as we know that the Republic outlaws slavery. At worst, the Syndicate would make deals with pirates/warlords who make use of slave labor on pirate planets, but even those would be special circumstances/something that only a few in the Syndicate would know of. Your average Syndicated Security captain isn't going to have orders to not fire on a pirate ship that's chasing a merchant captain through Syndicate space because "Oh wait, that's one we're making deals with."

Author: Alright, that sounds fair. Perhaps these sweatshops are run by the managers who are ruthless in trying to meet production numbers and need to pad their manufacturing lines to get that edge. I'll take a look at what the original author of the PR wrote for the next part and see how they showed the exhibition, since I think that is described. A convo that pops up saying that the Syndicate has dropped another "trusted provider" shouldn't be too hard though.

		government "Pirate"
		ship "Quicksilver" "Dog's Leash"
		ship "Quicksilver" "Burning Whip"
		ship "Quicksilver" "Iron Chains"

	on visit

> 		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

Reviewer 9: Passenger again.

mission "Adelita 5"

> 	name "Return Adelita to <planet>"
> 	description "Bring everyone back to <destination>. You will be paid <payment> after you complete the entire job."
> 	landing

Reviewer 9: ```suggestion
	landing
	name "Return Adelita to <planet>"
	description "Bring everyone back to <destination>. You will be paid <payment> after you complete the entire job."
```

			`	"It's more of a collage than an exact rendering," says the young woman you met in line, having followed you all this way. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet, for example. The magic of editing. What do you think?"`
			choice
				`	"Impressive."`
				`	"It's alright."`
				`	"Isn't that a bit deceptive?"`
					goto deceptive

			`	"I've been here ten times and I always manage to be awestruck," the woman replies. "Enjoy the exhibition while you still can!" She heads for the exit.`
			choice
				`	(Continue wandering around the exhibit.)`
					goto wander
				`	(Go to the exit.)`
					goto exit

			label deceptive

> 			`	"Deceptive?! No," the woman responds in a ruffled manner. "If you call this deception you must have a distaste for the media," she chuckles. "This form of journalism is as authentic as journalism gets! Now, of course Adelita take creative liberties, but it's all in the pursuit of education and justice."`

Author: ```suggestion
			`	"Deceptive?! No," the woman responds in a ruffled manner. "If you call this deception you must have a distaste for the media," she chuckles. "This form of journalism is as authentic as journalism gets! Now, of course Adelita take creative liberties, but it's all in the pursuit of education and justice.`
```

				die

			label "see us"
			`	"I have no idea," says Bunker. "But it's probably time to go."`
			choice
				`	"Call back the drones now."`
					goto "call drones"
				`	"I think we have to leave the drones behind."`

			label "no can do"
			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we'll lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
			choice
				`	"We're leaving. Now!"`
				`	"Fine. Get to the <ship> and lock down the drones as soon as they land. We take off on your mark."`
					goto "call drones"

			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. It'll take me no time to lock them down once we're on the <ship>, then we're out of here.`
			choice
				`	(Force him to stand down.)`
					goto pistol
				`	(Get back to the <ship> and let Bunker lock down the drones.)`

> 			`	You nod, and Bunker immediately packs up the entire controller and display set, packing it away in a briefcase in seconds.`

Author: ```suggestion
			`	You nod, and Bunker immediately packs up the entire controller and display set, packing it away in his briefcase in seconds.`
```

				goto flee

			label "call drones"

> 			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds." He packs up the entire controller and display set and has it packed away in a briefcase in seconds.`

Author: ```suggestion
			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds." He packs up the entire controller and display set and has it packed away in his briefcase in seconds.`
```

			label flee
			`	The both of you race onboard the <ship> and you prepare it for take off while Bunker heads to the cargo bay to receive the drones. "Lock down the drones as soon as they arrive and get belted in," you say over the comms to Bunker. "We'll take off on your mark."`
			`	Less than a minute later, you hear Bunker's reply. "Drones are locked down and ready to go."`

> 			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction; not far behind them, a Vanguard. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`

Author: ```suggestion
			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction; not far behind them, the Vanguard you saw earlier. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
```

				goto choice

			label political
			`	"Oh. Unfortunately for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installations," she says with a smile. She gets her smoothie to-go and walks out.`
				decline

			label choice

> 			`	"I head from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She gives a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, then pick up the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`

Reviewer 2: ```suggestion
			`	"I heard from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She gives a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, then pick up the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`
```
